pinot-common/pom.xml

```diff
@@ -107,6 +107,10 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
```

Comment:
Should the dependencies added here be in `test` scope ?

Reply:
It's already in `test` scope in parent pom:

```
     <!-- JMX exporter-->
      <dependency>
        <groupId>io.prometheus.jmx</groupId>
        <artifactId>jmx_prometheus_javaagent</artifactId>
        <version>0.19.0</version>
        <scope>test</scope>
      </dependency>
```